src/OpenSpaceToolkit/Physics/Data/Manifest.cpp

```diff
@@ -37,13 +39,26 @@ bool Manifest::isDefined() const
     return !dictionary_.isEmpty();
 }
 
+Instant Manifest::getLastUpdateTimestamp() const
```

Comment:
why not have the getter function name match the mem var that it is getting from?

Reply:
Great point. I've changed this in the MR that will update this one!